workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-article-template/ddm-template.ftl

```diff
@@ -1,90 +1,99 @@
 <#include "${templatesPath}/SVG">
-
 <script>
 	let href = window.location.href;
-
 	if (href.endsWith("/")) {
 		href = href.substring(0, href.length - 1);
 		window.location.assign(href);
 	}
 </script>
-
+<#assign
+	journalArticleId = .vars["reserved-article-id"].data
+	taxonomyCategoryBriefs = restClient.get("/headless-delivery/v1.0/sites/${groupId}/structured-contents/by-key/${journalArticleId}?nestedFields=embeddedTaxonomyCategory").taxonomyCategoryBriefs
+	taxonomyVocabularies = []
+/>
+<#list taxonomyCategoryBriefs as taxonomyCategoryBrief>
+<#assign taxonomyVocabularyName = taxonomyCategoryBrief.embeddedTaxonomyCategory.parentTaxonomyVocabulary.name />
+	<#if !taxonomyVocabularies?seq_contains(taxonomyVocabularyName)>
+		<#assign taxonomyVocabularies = taxonomyVocabularies + [taxonomyVocabularyName] />
+	</#if>
+</#list>
+<#assign taxonomyCategoriesMap = {} />
+<#list taxonomyCategoryBriefs as taxonomyCategoryBrief>
+<#assign taxonomyVocabularyName = taxonomyCategoryBrief.embeddedTaxonomyCategory.parentTaxonomyVocabulary.name />
```

Comment:
why do you make the same assign to the same variable 2 times in a row?
please just declare the variable out of the loop